.github/workflows/rust-test.yml

```
@@ -49,6 +49,34 @@ jobs:
       - name: Test
         run: cargo test --all-features
 
+  coverage:
+    name: Coverage
+    runs-on: "ubuntu-latest"
```

We pin the version of the runner so that update of `ubuntu-latest` to the next version doesn't break the pipelines (every update from GitHub has broken the pipelines so far)

```suggestion
    runs-on: "ubuntu-22.04"
```